### ISSUE TITLE
rancher2_app_v2 resource doesn't respect system default registry when updating app

### ISSUE THREAD

**Issue by dbason** [rancher/terraform-provider-rancher2#591]
On a Rancher cluster configured with a private default registry the rancher/shell:v0.1.5 image is trying to pull from public docker hub when the app is upgraded using terraform.  When upgraded using the UI the following values are passed to the API:
```
"values": {
    "global": {
        "systemDefaultRegistry": "registry.address:registry_port",
        "cattle": {
            "clusterId": "cluster_id",
            "clusterName": "cluster_name",
            "systemDefaultRegistry": "registry.address:registry_port"
        }
    }
}
```
The systemDefaultRegistry entries should be included in the global values by the provider so the default registry is respected.

gz#15233

**Comment by rawmind0** [comment]
Submitted PR https://github.com/rancher/terraform-provider-rancher2/pull/605 to fix the issue

**Comment by rawmind0** [comment]
Fix included at new Rancher2 tfp release v1.13.0
